src/features/callAssignments/components/CallAssignmentStatusCards.tsx

```diff
@@ -0,0 +1,114 @@
+import { useIntl } from 'react-intl';
+import { Card, Grid, List } from '@material-ui/core';
+
+import { CallAssignmentStats } from '../apiTypes';
+import StatusSectionHeader from './StatusSectionHeader';
+import StatusSectionItem from './StatusSectionItem';
```

Comment:
Love how this new code structure makes these components "local" to the feature, which means it matters less how they're named exactly.